org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeIsAliveTracker.java

```diff
@@ -62,8 +64,7 @@ public void removeHandler(ZigBeeThingHandler zigBeeThingHandler) {
      *
      * @param zigBeeThingHandler the {@link ZigBeeThingHandler}
      */
-    public void resetTimer(ZigBeeThingHandler zigBeeThingHandler) {
-        logger.debug("Reset timeout for handler with thingUID={}", zigBeeThingHandler.getThing().getUID());
```

Comment:
Any particular reason why you remove this debug line which helped you trying to find the issue?

Reply:
No - it was an error. I initially tried to consolidate the logging as there is a lot of logging around this and I think it's too much. I will look at that later though...